apps/api/src/movies/movies-search/movies-search.db.service.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { TestBed } from '@automock/jest'
+
+import { movieFactory } from '../../database/factories/movie'
+import { PrismaService } from '../../database/prisma.service'
+
+import { MoviesSearchDbService } from './movies-search.db.service'
+
+describe('MoviesSearchDbService', () => {
+  let service: MoviesSearchDbService
+
+  const queryRawMock = jest.fn()
+  const findManyMock = jest.fn()
+
+  beforeAll(async () => {
+    const { unit } = TestBed.create(MoviesSearchDbService)
+      .mock(PrismaService)
+      .using({
+        $queryRaw: queryRawMock.mockResolvedValue([]),
+        movie: {
+          findMany: findManyMock.mockResolvedValue([])
+        }
+      })
+      .compile()
+
+    service = unit
+  })
+
+  describe('findSimilarity', () => {
+    it('should run a raw query', async () => {
+      const title = 'Any Title'
+      await service.findSimilarity(title)
+
+      expect(queryRawMock).toHaveBeenCalledWith([
+        '\n      SELECT movie_id FROM "movie_titles" m\n      WHERE ',
+        " % ANY(STRING_TO_ARRAY(m.title, ' '))\n" +
+          '      ORDER BY similarity(t.title, ',
+        ') DESC\n      LIMIT ',
+        ';'
+      ], title, title, 10)
+    })
+  })
```

Comment:
Not sure if I get what you were trying to test here with the mock calls, I think you wanted to make sure the correct query is being run, my apologies if I misunderstood. I can revert this